qp_shotgun/qc_trim/__init__.py

```diff
@@ -0,0 +1,40 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaCommand
+
+from .qc_trim import qc_trim
+
+__all__ = ['qc_trim']
+
+# Define the kneaddata command
+req_params = {'input': ('artifact', ['per_sample_FASTQ'])}
+opt_params = {
+    'adapter': ['string', 'GATCGGAAGAGCACACGTCTGAACTCCAGTCAC'],  # 3' adapter
```

Comment:
Could you use better names (so they are easier to identify) for the users? For [example](https://github.com/qiita-spots/qp-deblur/blob/master/qp_deblur/__init__.py#L31)